modules/get-started/pages/licensing/overview.adoc

```diff
@@ -36,16 +36,13 @@ Starting with version 24.3, new Redpanda clusters automatically receive a trial
 
 To get a trial license key or extend your trial period, https://redpanda.com/try-enterprise[generate a new trial license key]. Or, https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^].
 
-[IMPORTANT]
-====
-Redpanda Connect does not accept trial license keys. To use enterprise features of Redpanda Connect, https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^] and then xref:redpanda-connect:get-started:licensing.adoc[apply your license key].
-====
-
 [NOTE]
 ====
 include::get-started:partial$licensing/block-upgrades-note.adoc[]
 ====
 
+In Redpanda Connect, you must https://redpanda.com/try-enterprise[generate] and xref:redpanda-connect:get-started:licensing.adoc#apply-a-license-key-to-redpanda-connect[apply a trial license key] to evaluate enterprise features. The 30-day evaluation period begins when you run your first data pipeline. After this period expires, you are blocked from using enterprise connectors unless you upgrade to an https://www.redpanda.com/upgrade[Enterprise Edition license^].
```

Comment:
We aren't able to track when "your first data pipeline" runs, this is why we initially didn't support trial licenses, so instead we're just allowing it to work until the trial license itself expires.

Reply:
Updated. Can you also check the information in this section for accuracy?

=== Redpanda Connect enterprise features